plugins/LilyPad/InputManager.cpp

```diff
@@ -433,7 +433,6 @@ Device *InputDeviceManager::GetActiveDevice(InitInfo *info, unsigned int *uid, i
 							  (devices[i]->oldVirtualControlState[j] > 31*FULLY_DOWN/32 && devices[i]->virtualControlState[j] < 7*FULLY_DOWN/8))) {
 									continue;
 						}
-						devices[i]->virtualControls[j].uid = devices[i]->virtualControls[j].uid;
```

Comment:
I tried looking into the code a bit but couldn't find any other scenarios where other variables would be useful. pinging @turtleli for feedback in case he has any ideas.